TOBIT REGRESSIONS

```diff
@@ -1,4 +1,17 @@
-# Tobit regression models with the specified conditions and variables as per the original analysis
+# Installing the libraries
+
+library(censReg)
+library(broom)
+library(dplyr)
+library(tidyr)
+
+# loading data
+df <- read_csv("D:/4 Sem/RR/data_DMDDG.csv")
```

Comment:
Could you please take a look at [this](https://github.com/DominikZabinski/rr_group_project/blob/main/supplementary.Rmd#L96-L102) part of code and change your so we can avoid reading files from local system and having hard-coded file paths? :)